invenio_curations/services/facets.py

```diff
@@ -15,7 +15,14 @@
     field="type",
     label=_("Type"),
     value_labels={
-        "curation": _("Curation review"),
+        "community-submission": _("Draft review"),
+        "community-inclusion": _("Community inclusion"),
+        "community-invitation": _("Member invitation"),
+        "guest-access-request": _("Guest access"),
+        "user-access-request": _("User access"),
+        "community-manage-record": _("Community manage record"),
+        "community-membership-request": _("Membership request"),
```

Comment:
Would it make sense to only include the types relevant for this package and update the `REQUESTS_FACETS` in `invenio.cfg` accordingly? Something like `REQUESTS_FACETS.update(INVENIO_CURATIONS_FACETS)` ?

Reply:
unfortunately, the relevant part of the `REQUESTS_FACET` config is not a dictionary but rather a `TermsFacet` class which only provides somewhat awkward access to the arguments it has been given.

i've updated the definition here to be dynamically based off the default values rather than a hard-coded list now.